### DISCUSSION TITLE
[Feature Request] Snackbar Addition

### DISCUSSION THREAD

**Discussion by Vanshbordia** [heroui-inc/heroui#1250] [Feature requests]
### Is your feature request related to a problem? Please describe.

NextUI currently has a tooltip that works in some cases, but a popup notification on the side or bottom like an actual snackbar that appears on the page

### Describe the solution you'd like

Ideally, a snackbar component can be passed a prop/invocation from click/ function, and that comes over other components on the z-index. Along with different locations like the bottom or top left/right of the page/container it is called in page/container

### Describe alternatives you've considered

https://m2.material.io/components/snackbars

https://material.angular.io/components/snack-bar/overview

https://mui.com/material-ui/react-snackbar/

**Comment by SellersEvan** [comment]
I 100% agree this should be a feature added to NextUI 🔥. I think it would be good to have options like stacking multiple items and persisting after route changing. Additionally, I am not sure if a Snackbar and Toast are different but I always preferred the name Toast 😎. I love how the [Vercel Toasts](https://vercel.com/design/toast) looks and works, they have the option to add action buttons and images.

### Screenshots
#### Vercel Toasts Collapse View
![image](https://github.com/nextui-org/nextui/assets/15301170/77b3d561-2a41-4e1d-b907-a4f7df00ea15)

 #### Vercel Toasts Expanded View (Hover)
![image](https://github.com/nextui-org/nextui/assets/15301170/b37a3fa7-a854-4a9d-a5ac-ddd548c062c3)
